dot/sync/syncer.go

```diff
@@ -469,14 +471,14 @@ func (s *Service) handleBlock(block *types.Block) error {
 		s.logger.Debug("imported block", "header", block.Header, "body", block.Body)
 	}
 
-	// TODO: if block is from the next epoch, increment epoch
-
 	// handle consensus digest for authority changes
 	if s.digestHandler != nil {
-		err = s.handleDigests(block.Header)
-		if err != nil {
-			return err
-		}
+		go func() {
+			err = s.handleDigests(block.Header)
+			if err != nil {
+				s.logger.Error("failed to handle block digest", "error", err)
+			}
+		}()
```

Comment:
Same here, can we just let the error do its thing?

Reply:
similar to below, if it errors it's not good but the node wouldn't crash or anything